### PR TITLE
feat: adds app tabs query to API

### PR DESCRIPTION
**Pull request by redreceipt**
test with `app-tabs` branch on the cluster

Screen Shot 2022-01-10 at 5 01 42 PM

### CODE REVIEW

File: packages/apollos-data-connector-rock/src/events/__tests__/__snapshots__/resolver.test.js.snap [hunk 1]
```diff
@@ -18,8 +18,8 @@ Object {
     ],
   },
   "errors": Array [
-    [GraphQLError: Cannot read property 'iCalendarContent' of undefined],
-    [GraphQLError: Cannot read property 'iCalendarContent' of undefined],
+    [GraphQLError: Cannot read properties of undefined (reading 'iCalendarContent')],
+    [GraphQLError: Cannot read properties of undefined (reading 'iCalendarContent')],
```
Comment (redreceipt) [inline packages/apollos-data-connector-rock/src/events/__tests__/__snapshots__/resolver.test.js.snap:22]:
```suggestion
    [GraphQLError: Cannot read property 'iCalendarContent' of undefined],
    [GraphQLError: Cannot read property 'iCalendarContent' of undefined],
```

File: packages/apollos-data-connector-postgres/src/features/resolver.js [hunk 2]
```diff
@@ -70,6 +71,16 @@ const resolver = {
     id: id('ScriptureFeature'),
   },
   Query: {
+    tabs: (_, args, { dataSources: { Feature } }) =>
+      ApollosConfig.APP_TABS.map(({ title, icon, features }) => ({
```
Comment (vinnyjth) [inline packages/apollos-data-connector-postgres/src/features/resolver.js:75]: What do you think about keeping this logic in the `dataSource`? It's not much, but it's a bit much for the resolver. 

Also - `ApollosConfig` in this context isn't cluster friendly (it won't pull a client's config). You need `dataSources.Config``

**Comment by vinnyjth** [comment]
Should this be pointed at `master` or `next`?

**Comment by redreceipt** [comment]
Yea I can move it to DS. I was gonna change the config when I merge in master to next branch. I don't wanna be blocked by next progress for this stuff.

**Comment by redreceipt** [comment]
ok @vinnyjth it's moved

**Review by vinnyjth — APPROVED** [review]

**Pull request merged**